website/content/api-docs/secret/ldap.mdx

```diff
@@ -71,6 +68,10 @@ to search and change entry passwords in LDAP.
 
 - `length` `(int: 64)` - The length of generated password strings. Note: some schemas may require shorter
   password lengths (such as `racf`). Mutually exclusive with `password_policy`
+- `connection_timeout` `(integer: 30 or string: "30s")` - Timeout, in seconds,
+  when attempting to connect to the LDAP server before trying the next URL in
+  the configuration. Note: Use request_timeout instead. When both `connection_timeout` and `request_timeout` are set,
+  the smaller value will be used for both connection and request timeouts.
```

Comment:
```suggestion
- `connection_timeout` `(integer: 30 or string: "30s")` - Timeout, in
   seconds, before trying the next URL in your Vault configuration when
   attempting to connect to your LDAP server. When both
   `connection_timeout` and `request_timeout` are set, Vault uses the
   smaller of the two values for connection **and** request timeouts.
```

Why should the reader use request timeout instead? If we want folks to use `request_timeout` over `connection_timeout` we should include a reason why

Reply:
It's now considered deprecated as a result of introducing the new ldap library. To somewhat preserve backwards compatibility, we will still to try honor the `connection_timeout`, but it's important to note that it may directly conflict with the `request_timeout` if they differ in value.